lib/browser/api/menu.js

```diff
@@ -39,6 +39,9 @@ Menu.prototype._init = function () {
         const found = this.groupsMap[id].find(item => item.checked) || null
         if (!found) v8Util.setHiddenValue(this.groupsMap[id][0], 'checked', true)
       }
+    },
+    menuClosed: () => {
+      this.emit('closed')
```

Comment:
Should this event be documented ?

Reply:
There are no events on the Menu object currently, so that's a good question. We could add other events tho, that would make this one a bit more useful too.

Reply:
I've added the documentation for the event anyway :)